pkg/daemon/config.go

```diff
@@ -176,31 +176,26 @@ func (output *ptp4lConf) populatePtp4lConf(config *string) error {
 	return nil
 }
 
-func getSource(is_ts2phc_master string) (source event.EventSource, err error) {
-	if master, err := strconv.ParseBool(strings.TrimSpace(is_ts2phc_master)); err == nil {
-		if master {
-			return event.GNSS, nil
-		} else {
-			return event.PPS, nil
+func getSource(is_ts2phc_master string) event.EventSource {
+	if ts2phcmaster, err := strconv.ParseBool(strings.TrimSpace(is_ts2phc_master)); err == nil {
+		if err != nil || ts2phcmaster {
```

Comment:
`err != nil` won't be true since it is in the path of  `err == nil` from previous line

Reply:
done